permissions/plugin-ant.yml

```diff
@@ -10,8 +10,7 @@ cd:
 developers:
   - "armfergom"
   - "jglick"
-  - "fcojfernandez"
```

Comment:
Is this related?

Reply:
It was suggested by the automation tooling i am using, but i am currently trying to find out why.  I suspect it was intentional - i will confirm.